Makefile

```diff
@@ -0,0 +1,104 @@
+# author: Sylvain Anderegg
+VERSION := $(shell uname -a)
+
+export VCS_URL:=$(shell git config --get remote.origin.url)
+export VCS_REF:=$(shell git rev-parse --short HEAD)
+export VCS_STATUS_CLIENT:=$(if $(shell git status -s),'modified/untracked','clean')
+export BUILD_DATE:=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+export SERVICES_VERSION=1.0.0
+export DOCKER_REGISTRY=itisfoundation
+
+## Tools ------------------------------------------------------------------------------------------------------
+#
+tools =
+
+ifeq ($(shell uname -s),Darwin)
+	SED = gsed
+else
+	SED = sed
+endif
+
+ifeq ($(shell which ${SED}),)
+	tools += $(SED)
+endif
+
+
+## ------------------------------------------------------------------------------------------------------
+.PHONY: all
+all: help info
+ifdef tools
+	$(error "Can't find tools:${tools}")
+endif
+
+
+## -------------------------------
+# Docker build and composition
+
+.PHONY: build build-devel
+# target: build, build-devel: – Builds all service images.
+build:
+	cd services && ${MAKE} build;
+
+build-devel:
+	cd services && ${MAKE} build-devel;
+
+
+.PHONY: up up-devel down
+# target: up, up-devel, down: – Starts/Stops services.
+up:
+	cd services && ${MAKE} up;
+
+up-devel:
+	cd services && ${MAKE} up-devel;
+
+down:
+	cd services && ${MAKE} down;
+
+.PHONY: push
+# target: push: – Pushes services to the registry.
+push:
+	cd services && ${MAKE} push;
+
+## -------------------------------
+# Tools
+
+.PHONY: info
+# target: info – Displays some parameters of makefile environments
+info:
+	@echo '+ VCS_* '
+	@echo '  - ULR                : ${VCS_URL}'
+	@echo '  - REF                : ${VCS_REF}'
+	@echo '  - (STATUS)REF_CLIENT : (${VCS_STATUS_CLIENT})'
+	@echo '+ BUILD_DATE           : ${BUILD_DATE}'
+	@echo '+ VERSION              : ${VERSION}'
+	@echo '+ DOCKER_REGISTRY      : ${DOCKER_REGISTRY}'
+	@echo '+ SERVICES_VERSION     : ${SERVICES_VERSION}'
+	@echo '+ PY_FILES             : $(shell echo $(PY_FILES) | wc -w) files'

```

Comment:
PY_FILES, SERVICES_VERSION , ... do not exist

Reply:
fixed.